main.c

```diff
@@ -380,12 +422,12 @@ void ledSet() {
  */
 void ledSetRow() {
   size_t bytesRead;
-  bytesRead = sdReadTimeout(&SD1, commandBuffer,
```

Comment:
Not sure if anyone is using this function. It seems to be buggy because the row needs `NUM_COLUMN` colors and each color is 3 bytes long. It was reading only `NUM_COLUMN * 2` bytes.

Reply:
ack good catch